README.md

```diff
@@ -1,90 +1,71 @@
+npm install @lokalise/frontegg-oauth-client
 ```
 
-## commitlint
-
-You can use `npm run commit` to interactively construct correct commit messsage.
-
-Check out [commitlint](https://commitlint.js.org) docs for examples of how to customise.
-
-## Release actions
-
-The following token needs to be set in the Github repo for the `prerelease` and `release` Github Actions to work:
-
-- `secrets.NPM_TOKEN` (need this to publish on NPM)
-
-When performing a release, make sure to follow our conventional commit approach, as described in [contribution documentation](https://github.com/lokalise/npm-package-template/blob/main/CONTRIBUTING.md).
-
-## License
-
-This project is APACHE, VERSION 2.0 licensed, see LICENSE.md for details.
-
-## Template Specific Internal Notes
-
-The following notes are only relevant to the maintainers of this template. You may delete this section when you clone it.
-
-### Creating a release of the template
-
-- Merge your PR and create a [new draft release](https://github.com/lokalise/npm-package-template/releases/new).
-- Create a new tag `vx.x.x`. Consider the type of changes you added. Major, minor or patch.
-- In Release title mention the same version `vx.x.x`.
-- Generate release notes.
-- Publish Release
-
-## Other Resources
+### Usage
+
+The library offers several APIs to initiate the OAuth flow and retrieve user data.
+
+```js
+// Create client instance
+const client = new FronteggOAuthClient({
+    baseUrl: 'https://frontegg-custom-url.com',
+    clientId: 'CLIENT_ID',
+    redirectUri: `${window.location.origin}/oauth/callback`,
+    logoutRedirectUri: window.location.origin,
+})
+
+try {
+    // Retrieve user
+    const user = await client.getUserData();
+
+    // Retrieve token 
+    const accessToken = user.accessToken;
+} catch (error) {
+    // In case we receive a 401 Unauthorized error, we need to redirect the user to the login page.
+    if (error instanceof FronteggError && error.status === 401) {
+        // Retrieve login URL
+        const loginUrl = await client.getOAuthLoginUrl() 
+        
+        // Redirect to Frontegg login page
+        window.location.href = loginUrl;
+        return;
+    }
+    
+    // Rethrow unknown error.
+    throw error
+}
+```
 
-This template represents the culmination of Lokalise technical recommendations as documented in the [Frontend Radar](https://lokalise.atlassian.net/l/cp/Bqkz2hC5).
+You also need to setup OAuth callback path in your browser app.
+
+```js
+// OAuth callback app - usually /oauth/callback URL
+const handleRequest = async (request: Request) => {
+    const oauthCode = new URL(request.url).searchParams.get('code')
+
+    if (!oauthCode) {
+        throw new Error('Missing oauth code');
+    }
+
+    // After successful login, the client can retrieve 
+    // the access token through the OAuth code.
+    await client.fetchAccessTokenByOAuthCode(oauthCode)
+    
+    
+    // Redirect the user to the main app
+    window.location.href = '/';
+}
+```
```

Comment:
As discussed, would be nice to have a credits section here.

Reply:
Good idea. Added.